num_params:  163781
BARRIER DEADLOCK DURING INITIALIZATION CREATING RANDOM POINT TO ALLEVIATE!
BARRIER DEADLOCK DURING INITIALIZATION CREATING RANDOM POINT TO ALLEVIATE!
Episode 1
Episode reward: 107.79999999999984
Episode steps: 81
Episode collisions: 0
Episode boundary: 0
Episode distance travelled: [6.33410965 6.91157831 5.45471416 6.07561862]

[Reward] Mean: 107.79999999999984, Standard Deviation: 0.0
[Steps] Mean: 81.0, Standard Deviation: 0.0
[Collisions] Mean: 0.0, Standard Deviation: 0.0
[Boundary] Mean: 0.0, Standard Deviation: 0.0
[Dist] Mean: [6.33410965 6.91157831 5.45471416 6.07561862], Standard Deviation: 0.5232368125296737
Exception ignored in: <module 'threading' from '/usr/lib/python3.8/threading.py'>
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 1388, in _shutdown
    lock.acquire()
KeyboardInterrupt: 
